src/content_script.js

```diff
@@ -74,6 +74,12 @@ const PASSIVE_SHARE_PATTERN_DETECTION_SELECTORS = [
   "[href*='facebook.com/sharer']", // Legacy Share dialog
 ];
 
+// Attributes distilled from selectors above. Update when necessary.
+const OBSERVER_ATTRIBUTES = [
+  "action", "aria-label", "class",
+  "data-action", "data-bfa-network", "data-destination", "data-login-with-facebook",
+  "data-oauthserver", "data-partner", "data-tag", "data-test-id", "data-tracking",
+  "href", "id", "title"];
```

Comment:
praise: This is so much cleaner to watch these attrs, rather the whole, dang DOM.